components/admin/actions.tsx

```diff
@@ -209,10 +213,11 @@ export default function Actions({ data }: PageProps): JSX.Element {
         band: values.band,
         reason: values.reason,
         biddingNumber: values.biddingNumber,
+        generateBiddingNumber: values.biddingNumberType === 'generate',
       },
     };
     updateApplication(request);
-    router.reload();
+    _.delay(() => router.reload(), 500);
```

Comment:
You don't need lodash for this...
```
setTimeout(() => router.reload(), 500)
```

Although this kind of thing is usually a code smell. What's taking 500ms that you need to wait for?

Reply:
Yep agree, it was added due to the page reloading the data before it's persisted.